.github/workflows/pr.yml

```diff
@@ -4,47 +4,58 @@ on: [pull_request]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
         node-version: [12.x]
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
wonder why it worked without the 2 spaces :/

Reply:
It's just the formatter. Both are valid yaml but the default setting of VS Code idents it.